docs/src/man/getting_started.md

```diff
@@ -673,6 +673,46 @@ julia> Tables.rowtable(df)
  (x1 = 1, x2 = 3, y = 5)
  (x1 = 2, x2 = 4, y = 6)
 ```
+Now assume that a data frame `df` contains predictions from a model producing scores
+for three levels `a`, `b` and `c` of a nominar target variable.
+For each row we want to get the level with the highest score.
```

Comment:
I'd avoid the context, finding the column with the highest value can happen in many situations, no need to justify.
```suggestion
With this approach, we can easily compute for each row the name of column with the highest score:
```